[ES] Add Mappers_Guild (#8475)
ppy/osu-wiki: wiki/Community/Mappers_Guild/es.md

> ---
> tags:
>   - Mapper's Guild
>   - MG
> ---
> 
> # Mappers' Guild
> 
> ![](img/mg-logo.png)
> 

Reviewer: ```suggestion
---
tags:
  - Mapper's Guild
  - MG
---

# Mappers' Guild

![](img/mg-logo.png)

```

Author: Is there any way you could DM me this changes? Github doesn't let me apply the changes since there are no actual changes to the lines themselves

> **Mappers' Guild** es un grupo de mappers responsables de crear [beatmaps](/wiki/Beatmap) para los [artistas destacados](/wiki/Featured_Artists) de osu!. Utilizando un formato de gremios estilo MMO, permite que los mappers ganen recompensas por participar en el proceso de clasificación de beatmaps de artistas destacados y cooperar en misiones relacionadas con mapping.

Reviewer: ```suggestion
**Mappers' Guild** es un grupo de mappers responsables de crear [beatmaps](/wiki/Beatmap) para los [artistas destacados](/wiki/Featured_Artists) de osu!. Utilizando un formato de gremios estilo MMO, permite a los mappers ganar recompensas por participar en el proceso de clasificación de beatmaps de artistas destacados y cooperar en misiones relacionadas con mapping.
```

La organización y discusión del gremio puede ser encontrada en el canal de discord `#mappers-guild` en el server de [osu!dev Discord server](https://discord.gg/ppy).

## Sistema

> *Para la información mas actualizada sobre el Mappers' Guild, referirse a las [preguntas frecuentes de este sitio](https://mappersguild.com/faq).*
> 
> Mappers con 3 o más mapas clasificados pueden unirse al gremio en cualquier momento a través de [Mappers' Guild website](https://mappersguild.com/) autenticando su cuenta de osu!. Cualquiera es libre de ver las misiones y actividades de mappeo en la página web.
> 
> Los mappers pueden conseguir recompensas participando en el proceso de clasificación de mapas que sean de Artistas Destacados y estén conectados al Mappers' Guild. Esto se enfoca principalmente en la creación de beatmaps, sin embargo, las recompensas también son proveídas a usuarios que contribuyan al [moddeo](/wiki/Modding), a la creación de [storyboards](/wiki/Storyboard), y completando misiones dentro del gremio.

Reviewer: ```suggestion
*Para obtener la información más actualizada sobre el sistema de Mappers' Guild, consulte el [FAQ de su sitio web](https://mappersguild.com/faq).*

Los mappers con 3 o más mapas clasificados pueden unirse al gremio en cualquier momento a través del [sitio web de Mappers' Guild](https://mappersguild.com/) al autenticar su cuenta osu!. Cualquier otra persona es libre de ver las actividades de búsqueda y mapping del sitio web.

Los mappers pueden obtener recompensas al participar en el proceso de clasificación de los mapas de artistas destacados que esten conectados con Mappers' Guild. Esto se enfoca principalmente en la creación de beatmaps, sin embargo, también se brindan recompensas para los usuarios que contribuyan [modeando](/wiki/Modding), creando [storyboards](/wiki/Storyboard), y completando misiones dentro del gremio.
```

### Bandas

> Las bandas son grupos de mappers que se registran para completar misiones juntos. Los mappers se pueden unir a cuantas bandas quieran en cualquier momento dado.
> 
> Misiones especiales requieren un rango mayor de gremio para participar. Para una banda calificar como una de alto rango, al menos la mitad de sus miembros deben de ser del rango requerido o más alto.

Reviewer: ```suggestion
Las bandas son grupos de mappers que se registran para completar misiones juntos. Los mappers pueden unirse a tantos bandas como quieran a la vez.

Las misiones especiales requieren rangos de gremio más altos para participar. Para que una banda califique como un rango más alto, al menos la mitad de sus miembros deben tener el rango requerido o superior.
```

Author: did apply this fully but changed "tantos" to "tantas"

Reviewer: correct

### Misiones

> Las misiones recompensan a los mapper por seguir ciertos requerimientos cuando crean sus mapas de los Artistas Destacados. Usualmente, misiones que involucren la creación de cierta cantidad de beatmaps para Artistas Destacados nuevos, sin embargo, algunos involucran trucos pequeños.
> 
> Cada misión tiene su propia cantidad de personas en la banda/especificaciones de clasificación, recompensas por aceptar, y plazo. Todos los miembros de una banda deben pagar el precio listado para aceptar una misión. Los precios son un usualmente más altos para misiones que involucren trucos y en solitario.

Reviewer: ```suggestion
Las misiones recompensan a los mappers por seguir ciertos requisitos al crear sus mapas de artistas destacados. Por lo general, las misiones implican la creación de una cierta cantidad de beatmaps para un nuevo artista destacado, sin embargo, algunas misiones implican pequeños trucos.

Cada misión tiene sus propias especificaciones de tamaño/rango de banda, precio a aceptar y fecha límite. Todos los miembros de una banda deben pagar el precio indicado para aceptar la misión. Los precios suelen ser más altos para misiones ingeniosas y misiones en solitario.
```

> Las misiones expiran automáticamente después de 3 meses de inactividad. Cualquier usuario puede reabrir la misión usando sus puntos del Mappers' Guild de su banda.
> 
> La participación de las misiones no es requerida para miembros del Mappers' Guild. Si un usuario quiere, puede crear mapas solo para recibir recompensas.

Reviewer: ```suggestion
Las misiones caducan automáticamente después de 3 meses de inactividad. Cualquier usuario puede reabrir misiones gastando su reserva disponible de puntos del Mappers' Guild.

No se requiere la participación en misiones para los miembros del Mappers' Guild. Si un usuario quiere, puede crear beatmaps solo para ganar recompensas.
```

### Recompensas

> ![](img/mg-badges.png "The four tiers of the Mappers' Guild profile badge")

Reviewer: ```suggestion
![](img/mg-badges.png "Los cuatro niveles de la insignia de perfil del Mappers' Guild")
```

> Las recompensas son distribuidas a través de un sistema de puntos. Cuando un usuario alcanza una cantidad de puntos específica, ganan un rango diferente significado por una insignia de perfil. Existen cuatro rangos de insignias de perfil en 100, 250, 500, 1000 puntos respectivamente.
> 
> Las recompensas son basadas en los puntos totales de un usuario. Los puntos disponibles de un usuario son una subsección de su total de puntos siguiendo el mismo modelo que los [kudosu](/wiki/Modding/Kudosu) de osu! con "total obtenido" y "disponible" en sus respectivas secciones.

Reviewer: ```suggestion
Las recompensas se distribuyen a través de un sistema de puntos. Cuando un usuario alcanza una cantidad de puntos específica, gana un rango diferente indicado por una insignia de perfil. Hay cuatro niveles de insignias de perfil en 100, 250, 500 y 1000 puntos respectivamente.

Las recompensas se basan en los puntos totales de un usuario. Los puntos disponibles de un usuario son una subsección de sus puntos totales siguiendo el mismo modelo que los fondos de [kudosu](/wiki/Modding/Kudosu) de osu! "total obtenido" y "disponible".
```

> Los puntos se obtienen al hacer diferentes tareas relacionadas al mappeo. Las ganancias pueden variar entre tareas, aunque, son generalmente sopesados de acuerdo a la cantidad de tiempo que se requiere. Por ejemplo, mappear una dificultad `Hard` toma más tiempo que mappear una dificultad `Easy`, entonces la primera da más puntos que la segunda.
> 
> Las tareas de creación de dificultad recompensan a un usuario más puntos si el mapa está relacionado a una misión. Completar una misión antes de su plazo, provee a los usuarios con puntos de bono por cada dificultad mappeada y un bono fijo por todos los usuarios involucrados en la misión. Este bono fijo es perdido cuando la misión termina su plazo, sin embargo, los bonos de las dificultades se mantienen con devoluciones decaídas luego de 0, 20, y 40 días de incremento luego de su plazo.

Reviewer: ```suggestion
Los puntos se ganan por diferentes tareas relacionadas con el mappeo. Las ganancias varían entre tareas, aunque generalmente se ponderan de acuerdo con la cantidad de tiempo requerido. Por ejemplo, mapear una dificultad `Hard` lleva más tiempo que mapear una dificultad `Easy`, por lo que la primera otorga más puntos que la segunda.

Las tareas de creación de dificultad otorgan al usuario más puntos si el beatmap está asociado con una misión. Completar una misión antes de su fecha límite proporciona a los usuarios puntos de bonificación por cada dificultad mapeada y una bonificación fija para todos los usuarios involucrados en la misión. Esta bonificación fija se pierde después de que se pasa la fecha límite, sin embargo, las bonificaciones de dificultad se mantienen con rendimientos decrecientes después de incrementos de 0, 20 y 40 días después de la fecha límite.
```

> ### Beatmaps de muestra
> 
> *Articulo principal: [beatmaps de muestra de los Artistas Destacados](/wiki/Featured_Artists/Featured_Artist_showcase_beatmaps)*
> 
> Los miembros del Mappers' Guild también pueden participar en la creación de beatmaps de muestra para los anuncios de los Artistas Destacados en el futuro. Al momento del anuncio del Artista Destacado correspondiente, estos mapas serán resaltados en el [puesto de noticias](https://osu.ppy.sh/home/news) y usualmente son promovidos en el [@osugame Twitter](https://twitter.com/osugame) también.

Reviewer: ```suggestion
### Beatmaps de exhibición

*Artículo principal: [Beatmaps de exhibición de artistas destacados](/wiki/Featured_Artists/Featured_Artist_showcase_beatmaps)*

Los miembros de Mappers' Guild también pueden participar en la creación de beatmaps de exhibición para futuros anuncios de artistas destacados. Tras el anuncio del artista destacado correspondiente, estos beatmaps se destacan en la [publicación de noticias](https://osu.ppy.sh/home/news) y, a menudo, también se promocionan en el [Twitter de @osugame](https://twitter.com/osugame).
```

> ### Concursos de mappeo
> 
> El Mappers' Guild tambien es usado para manejar los concursos de mappeo. Desde enero del 2020, todas las iteraciones del [Concurso de Mappeo Mensual](/wiki/Contests/Monthly_Beatmapping_Contest) han sido alojadas, examinadas, y juzgadas usando el sistema provisto por el Mappers' Guild. Cualquier usuario puede alojar concursos a través del Mappers' Guild para más accesibilidad a la comunidad de mappeo de osu!.

Reviewer: ```suggestion
### Concursos de beatmapping

El Mappers' Guild también se utiliza para administrar concursos de beatmapping. Desde enero de 2020, todas las iteraciones del [Concurso mensual de beatmapping](/wiki/Contests/Monthly_Beatmapping_Contest) se han organizado, proyectado y juzgado mediante sistemas proporcionados por Mappers' Guild. Cualquier usuario puede organizar concursos a través de Mappers' Guild para aumentar la accesibilidad a la comunidad de mapping de osu!.
```

## Historia

> El Mappers' guild se lanzó con un grupo de menos de 10 mappers a finales del 2017. Dada la información sobre un Artista Destacado en ese tiempo, [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19), se les dio la tarea de crear seis beatmaps para acompañar el [anuncio de noticias](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) del artista.
> 
> Para promover los Artistas Destacados más viejos, el grupo empezó a incrementar su cantidad de miembros y clasifico 8 beatmaps de artistas diferentes. Estos beatmaps fueron publicados en un [anuncio de noticas](https://osu.ppy.sh/home/news/2018-03-17-new-mappers-guild-pack-release) a la mano un [paquete de beatmaps](/wiki/Beatmap/Packs) descargable.
> 
> Siguiendo los pasos de estos paquetes de beatmaps, el Mappers' Guild incremento su cantidad de miembros continuamente mientras publicaba paquetes para los anuncios de Artistas Destacados, como [High Tea Music](https://osu.ppy.sh/home/news/2018-08-13-new-featured-artist-high-tea-music), [Culprate](https://osu.ppy.sh/home/news/2018-11-21-new-featured-artist-culprate), y [HyuN](https://osu.ppy.sh/beatmaps/artists/33), acompañado de otros paquetes de canciones de Artistas Destacados viejos.

Reviewer: ```suggestion
El Mappers' Guild se lanzó con un grupo de menos de 10 mappers a fines de 2017. Dada la información sobre un próximo artista destacado en ese momento, [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19), se les encomendó la tarea de crear seis beatmaps para acompañar el [anuncio de la noticia](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) del artista.

Para promocionar a los artistas destacados más antiguos, el grupo aumentó su número de miembros y clasificó 8 beatmaps de diferentes artistas. Estos beatmaps se publicaron en una [noticia](https://osu.ppy.sh/home/news/2018-03-17-new-mappers-guild-pack-release) junto con un [paquete de beatmaps](/wiki/Beatmap/Packs) descargable.

Siguiendo los pasos de estos dos paquetes de beatmaps, el Mappers' Guild aumentó continuamente su número de miembros mientras publicaba paquetes para anuncios de nuevos artistas destacados, como [High Tea Music](https://osu.ppy.sh/home/news/2018-08-13-new-featured-artist-high-tea-music), [Culprate](https://osu.ppy.sh/home/news/2018-11-21-new-featured-artist-culprate), y [HyuN](https://osu.ppy.sh/beatmaps/artists/33), acompañados de otros paquetes para canciones de artistas destacados más antiguas.
```

### Automatización

> Mientras que el plan inicial del Mappers' Guild siempre ha sido tener elementos de un MMO como misiones, fue encontrado impráctico para manejar mientras a la vez se manejaban plazos para la cantidad limitada de miembros participantes del gremio. ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) y ::{ flag=CL }:: [Milan-](https://osu.ppy.sh/users/1052994) empezaron con el desarrollo de una página web para automatizar el Mappers' Guild a finales del 2018, permitiendo una distribución de recompensas y mantenimiento del sistema de misiones/bandas más adecuado.
> 
> La [página web del Mappers' Guild](https://mappersguild.com/) fue lanzada el primero de marzo del 2019, y es donde todo lo relacionado al Mappers' Guild es controlado. Desde el inicio abierto para el público, el gremio a alcanzado una cantidad de más de 500 miembros y publica una [actualización del puestos de noticias](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) para mantener al resto de la comunidad de osu! al tanto de los concursos de mappeo de Artistas Destacados.
> 

Reviewer: ```suggestion
Si bien el plan inicial para Mappers' Guild siempre había sido tener elementos MMO como misiones, se descubrió que eso no era práctico de manejar mientras se administraban manualmente los plazos para el número limitado de participantes del gremio. ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) y ::{ flag=CL }:: [Milan-](https://osu.ppy.sh/users/1052994) comenzaron a desarrollar un sitio web para automatizar Mappers' Guild a finales de 2018, lo que permite una distribución más adecuada de las recompensas y la gestión de los sistemas de misiones/bandas.

El [sitio web de Mappers' Guild](https://mappersguild.com/) se lanzó el 1 de marzo de 2019 y es donde actualmente se controla todo lo relacionado con Mappers' Guild. Desde que se abrió al público, el gremio ha alcanzado un número de miembros de más de 500 y publica [noticias actualizadas](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) para mantener al resto de la comunidad de osu! al día con el contenido de mapas de artistas destacados.

```
